cmd/up/globals/globals.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 Upbound Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package globals
+
+// Allowed values for the global format option
+const (
+	OutputJSON = "json"
+	OutputYAML = "yaml"
+)
```

Comment:
Suggest something like the following. Defining a type means that any subcommands can receive it (similar to what you are doing currently with `globals.Globals`.
```suggestion
// Format is the structure in which information is emitted to the terminal.
type Format string

// Allowed values for the global format option.
const (
        Default Format = "default"
	JSON Format = "json"
	YAML Format = "yaml"
)
```